templates/entry/single.html.twig

```diff
@@ -37,7 +37,7 @@
         {{ component('entries_cross', {entry: entry}) }}
         {% include 'layout/_flash.html.twig' %}
         {% include 'entry/comment/_options.html.twig' %}
-        {% include('user/_visibility_info.html.twig') %}
+        {% include('user/_muted_info.html.twig') %}
         
         {% if user is defined and user and user.visibility is same as 'visible' and (user_settings.comment_reply_position == constant('App\\Controller\\User\\ThemeSettingsController::TOP')) %}
```

Comment:
all of these likely need to change to check muted instead now

I started to collect a list but it's easily dozens of places in https://github.com/MbinOrg/mbin/pull/212/files#diff-4474cfa4c5acf2dcb955841f0e57b248d4ee61e695467130752cf785660433a0

basically everywhere `user.visibility is same as 'visible'` appears in twig templates